### PR TITLE
Add Playwright e2e tests 🎭

### PR DESCRIPTION
**Pull request by MarcusFelling**
Added 7 [Playwright ](https://playwright.dev/) e2e test cases that are executed in "podcast web CI/CD workflow", across chromium, firefox, and webkit. Base URL of the site under test is set via env variable `BASEURL`. For GHA, this value is constructed using `WEBAPP_NAME` variable.

GHA is configured to generate a test job summary and upload artifacts for HTML reports with video and trace files that can be downloaded to assist with post-mortem debugging.

Example failed test GHA run here: https://github.com/MarcusFelling/dotnet-podcasts/actions/runs/3055457978/attempts/1#summary-8358350472

### CODE REVIEW

**Comment by MarcusFelling** [comment]
@jamesmontemagno  - looks like creds are bad for az login.

**Comment by jamesmontemagno** [comment]
Yeah that is fine @MarcusFelling  don't worry about it

File: src/Web/E2E/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "version": "1.0.0",
+  "description": "---\r page_type: sample\r description: \".NET 6 reference application shown at .NET Conf 2021 featuring ASP.NET Core, Blazor, .NET MAUI, Microservices, and more!\"\r languages:\r - csharp\r products:\r - dotnet-core\r - ef-core\r - blazor\r - dotnet-maui\r - azure-sql-database\r - azure-storage\r - azure-container-apps\r - azure-container-registry\r - azure-app-service-web\r ---",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {},
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/MarcusFelling/dotnet-podcasts.git"
```
Comment (nishanil) [inline src/Web/E2E/package.json:12]: I am not sure how this works, but should the url be pointing to the main repo instead of the fork?

Comment (MarcusFelling) [reply src/Web/E2E/package.json:12]: Nice catch! Yes, it should be pointing to main repo.

Comment (MarcusFelling) [reply src/Web/E2E/package.json]: Fixed

**Pull request closed without merging**